torch/csrc/jit/script/sugared_value.cpp

```diff
@@ -14,6 +14,81 @@ struct NoneValue : SugaredValue {
   }
 };
 
+std::shared_ptr<SugaredValue> MethodValue::call(
+    const SourceRange& loc,
+    Function& caller,
+    at::ArrayRef<NamedValue> inputs,
+    at::ArrayRef<NamedValue> attributes,
+    size_t n_binders) {
+
+  Graph& graph = *caller.graph();
+  auto& callee = *method_;
+  try {
+    callee.ensure_defined();
+  } catch (std::exception&) {
+    throw ErrorReport(loc)
+        << " method '" << callee.name()
+        << "' is called recursively involving this call site. Recursive calls are not supported";
+  }
+
+  auto callee_graph = callee.graph();
+  auto& schema = callee.getSchema();
+
+  std::vector<NamedValue> inputsWithSelf(inputs.begin(), inputs.end());
+  auto self_val = self_.value(*caller.graph());
+  if (self_val)
+  {
+    inputsWithSelf.insert(inputsWithSelf.begin(), self_val);
+  }
+
+  std::stringstream failure_messages;
+  auto matched_schema = tryMatchSchema(
+      schema,
+      loc,
+      graph,
+      c10::nullopt,
+      inputsWithSelf,
+      attributes,
+      failure_messages,
+      /*conv_tensors_to_nums=*/true);
+
+
+  if (!matched_schema)
+  {
+    throw ErrorReport(loc)
+        << " mismatch between arguments and parameters " << failure_messages.str();
+  }
+
+  if (matched_schema.value().return_types.size() > 1)
+  {
+    throw ErrorReport(loc)
+        << "multiple returns aren't yet supported";
+  }
+
+  auto fun_constant = graph.create(prim::Constant);
```

Comment:
Code to correctly construct CallFunction should be in ir.h:

```
::createFunctionCall(Value* fn, at::ArrayRef inputs);
```

Reply:
will fix